log/logger.go

```diff
@@ -8,52 +8,58 @@ import (
 // Logger is an optional custom logger.
 var Logger *log.Logger
 
+func initDefaultLogger() {
```

Comment:
Why not move this into the main body instead? It still gives importing packages the ability to override it, and it stays more consistent to how the base logger for Go actually is implemented.

```go
var Logger = log.New(os.Stderr, "", log.LstdFlags)
```